terra2/assetlist.json

```diff
@@ -459,6 +459,34 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/amproar.png"
         }
       ]
+    },
+    {
+      "description": "Local Money - the decentralized on/off ramp.",
+      "type_asset": "cw20",
+      "address": "terra1xzkel96e5e8vfmqw7valzdzzv9hqasfyslclvnmvxdejvpda3xwsskxzus",
+      "denom_units": [
+        {
+          "denom": "cw20:terra1xzkel96e5e8vfmqw7valzdzzv9hqasfyslclvnmvxdejvpda3xwsskxzus",
+          "exponent": 0
+        },
+        {
+          "denom": "whLOCAL",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:terra1xzkel96e5e8vfmqw7valzdzzv9hqasfyslclvnmvxdejvpda3xwsskxzus",
+      "name": "whLOCAL",
+      "display": "whLOCAL",
+      "symbol": "whLOCAL",
```

Comment:
is this being bridged from Kujira via Wormhole?

Reply:
This is $LOCAL minted on Terra Classic and moved to Terra2 via Wormhole.